packages/pkg-utils/lib/get-changed-packages.js

```diff
@@ -21,27 +21,26 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
  * SOFTWARE.
  */
-import { runCommandSync } from '@instructure/command-utils'
-import path from 'path'
-import { getPackages } from './get-packages'
 
-import type { Package } from './get-packages'
+const path = require('path')
+const getPackages = require('./get-packages')
+const childProcess = require('child_process')
```

Comment:
`pkg-utils` needed to go back to CommonJS because its used by a lot of other packages that are written in CommonJS. if I'd rewrite this to ES6 import id need to rewrite lots of others too